### DISCUSSION TITLE
Need help deployment

### DISCUSSION THREAD

**Discussion by vardankumar** [vercel/community#1632] [Help]
### Summary

hello i am new to vercel. I just deployed a project on vercel but its not showing properly. can anybody take some time to help me out with this.

https://weather-ocvc2oh5x-vardankumar.vercel.app/

**Comment by swarnava** [comment]
Hi, do you have a [minimal repository](https://github.com/evedriver/minimal-reproducable) which we can look into?

**Reply by vardankumar** [reply]
https://github.com/vardankumar/Weather-App

**Comment by Pieparker** [comment]
Looks like you may be using Vite. There's a heap of resources out there for white screens when deploying Vite/React which you may wish to check out! I believe this will be the most useful to you. In short, you need to ensure that your base directory is configured correctly.

- https://stackoverflow.com/questions/72005194/vue-3-vite-built-application-shows-blank-page
